### PR TITLE
[SVCHOST] Fix PSID initialization

### PR DESCRIPTION
**Pull request by learn-more**
JIRA issue: [CORE-17151](https://jira.reactos.org/browse/CORE-17151)

### CODE REVIEW

File: base/services/svchost/globals.c [hunk 1]
```diff
@@ -274,13 +274,18 @@ ScCreateWellKnownSids (
     for (i = 0; i < RTL_NUMBER_OF(SidData); i++)
     {
         /* Convert our optimized structure into an actual SID */
-        Status = ScAllocateAndInitializeSid(&SidData[i].Sid,
+        Status = ScAllocateAndInitializeSid(SidData[i].Sid,
                                             &SidData[i].Authority,
                                             1);
-        if (!NT_SUCCESS(Status)) break;
+
```
Comment (HBelusca) [inline base/services/svchost/globals.c:280]:
```suggestion
```

File: base/services/svchost/globals.c [hunk 2]
```diff
@@ -234,7 +234,7 @@ ScDomainIdToSid (
 NTSTATUS
 NTAPI
 ScAllocateAndInitializeSid (
-    _Out_ PVOID *Sid,
+    _Out_ PSID *Sid,
```
Comment (HBelusca) [inline base/services/svchost/globals.c:237]:
```suggestion
    _Out_ PSID* Sid,
```
(keeping "consistency" with the adjustments done in the structures below)

Comment (learn-more) [reply base/services/svchost/globals.c:237]: no.

File: base/services/svchost/globals.c (hunk 1 above)
Comment (learn-more) [reply base/services/svchost/globals.c:280]: no.

**Pull request merged**